hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCLI.java

```diff
@@ -57,8 +55,14 @@ public void testMkdirRootNonExistentContainer() throws Exception {
 
     String nonExistentContainer = "nonexistent-" + UUID.randomUUID();
 
+    // Disable creation of new filesystem
+    final AbfsConfiguration conf = getConfiguration();
+    conf.setBoolean(AZURE_CREATE_REMOTE_FILESYSTEM_DURING_INITIALIZATION,
+        false);
```

Comment:
Creating copy of Configuration (like `new Configuration(getRawConfiguration())`) would be better than updating existing AbfsConfiguration . The FsShell could be created using the updated copy in that case.

Reply:
Updated the rawConfiguration as suggested.